docs/lib-documentation.md

```diff
@@ -120,7 +96,7 @@ Kurtosis uses a Docker volume to keep track of test state, and needs to mount th
 The set of ports that the container will be listening on, in the format `NUM/PROTOCOL` (e.g. `80/tcp`, `9090/udp`, etc.).
 
 ### Map\<String, Func(File)\> fileGeneratingFuncs
-Declares the files that will be generated before your service starts and made available on the container's filesystem, as well as the logic for generating their contents. The file keys declared here (which can be any string you like) will be the same keys used to identify the files in the map arg to [ContainerConfigFactory.getRunConfig][containerconfigfactory_getrunconfig].
+Declares the files that will be generated before your service starts and made available on the container's filesystem, as well as the logic for generating their contents. The file keys declared here (which can be any string you like) will be the same keys used to identify the files in the map arg to [NetworkContext.addServiceToPartition][NetworkContext_addServiceToPartition].
```

Comment:
I _think_ casing is important here - the linkref is actually `networkcontext_addservicetopartition` (so not camel-cased)